Confus/Confus/Player.h

```diff
@@ -27,8 +30,12 @@ namespace Confus {
     public:
         irr::scene::IAnimatedMeshSceneNode* PlayerNode;
         irr::scene::ICameraSceneNode* CameraNode;
+
+		Health m_PlayerHealth;
```

Comment:
Can you remove the unneeded white lines? Also, since this is public, the guidelines say you should ommit the m_ prefix